frontend/src/stories/components/Leaderboard/LeaderboardTable.stories.js

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Please implement the storybook entry for this component

Reply:
I had written this PR with the intent that another person from my group would write the fixtures and storybook and would merge it into this code afterwards. I will just merge their branch in, since they already implemented it in PR #73

Reply:
Got it!